app/models/order.py

```diff
@@ -109,6 +109,13 @@ def __str__(self):
     def get_invoice_number(self):
         return 'O' + str(int(time.mktime(self.created_at.timetuple()))) + '-' + str(self.id)
 
+    @staticmethod
+    def get_updatable_fields(cls):
```

Comment:
`cls` is unused, right? Also wouldn't be a public constant better here (as this has to be evaluated on every call)

Reply:
This is exactly what I did earlier but I was told to add it inside the model class. The point about getting evaluated on every call never occured to me then otherwise I would have countered that argument. Thanks, will move it outside the model class.